site/guide/organize-model-inventory.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Organize models in the inventory"
+---
+
+Customize the model inventory to fit your needs.
+
+## Prerequisites
+
+Any user can complete these steps.
```

Comment:
If there are no prerequisites, you could just comment out this section. In this case, having at least some models registered in the inventory might be a requirement.